addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -88,7 +88,6 @@ odoo.define('website_slides.quiz', function (require) {
             var self = this;
             return this._super.apply(this, arguments).then(function ()  {
                 self._renderAnswers();
-                self._renderAnswersHighlighting();
```

Comment:
What was the purpose of this line ?

Reply:
+1: why removing this ? are yo usure there is no side effect ?

Reply:
I talked about that with @qmo-odoo and he told me it was used before it was included directly in the template.
I tested it and it doesn't seem to break anything

Reply:
Ok, thanks !